aroquev/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -55,13 +67,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // TODO: verify that message is not empty
+    String username = request.getParameter("comments-username-input");
     String comment = request.getParameter("comments-body-input");
     long timestamp = System.currentTimeMillis();
 
     Entity commentEntity = new Entity("Comment");
```

Comment:
Comment class could also have a "toEntity" method to allow putting into Datastore.

Reply:
+1

Reply:
Okay so this method would mean that I have a Comment object which can call this toEntity() method and return an Entity that represents that Comment right?